apps/web/src/styles.css

```
@@ -243,36 +297,1443 @@ body {
+}
+
+.clean-faq__marker::before,
+.clean-faq__marker::after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 0.75rem;
+  height: 1.5px;
+  border-radius: 999px;
+  background: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)